container-chains/templates/simple/runtime/src/lib.rs

```diff
@@ -170,9 +170,7 @@ pub mod opaque {
 }
 
 impl_opaque_keys! {
-    pub struct SessionKeys {
-        pub aura: Aura,
-    }
+    pub struct SessionKeys { }
```

Comment:
do we need this anymore?

Reply:
This is being referenced in the impl block

```rust
impl sp_session::SessionKeys<Block> for Runtime {
        fn generate_session_keys(seed: Option<Vec<u8>>) -> Vec<u8> {
            SessionKeys::generate(seed)
        }

        fn decode_session_keys(
            encoded: Vec<u8>,
        ) -> Option<Vec<(Vec<u8>, sp_core::crypto::KeyTypeId)>> {
            SessionKeys::decode_into_raw_public_keys(&encoded)
        }
    }
```

Reply:
@tmpolaczyk suggested we could leave it !unimlemented() and pass parachain_config.dev_key_seed = None. But I dont really mind, we can merge the PR as is

Reply:
I'm fine either way, but we should add a comment saying that this is intentionally empty, because it is only required to satisfy a trait bound of `sc_service::spawn_tasks`, and session keys are not used in container chains.